### ISSUE TITLE
CVE-2024-9680 / Mozilla Security Advisory 2024-51

### ISSUE THREAD

**Issue by tmo1** [mullvad/mullvad-browser#333]
Hello,

As far as I can tell, Mullvad Browser is vulnerable to this bug, which Mozilla calls critical:

https://www.mozilla.org/en-US/security/advisories/mfsa2024-51/
https://www.cve.org/CVERecord?id=CVE-2024-9680

**Comment by hackademix** [comment]
No, [Mullvad Browser 13.5.7](https://github.com/mullvad/mullvad-browser/releases/tag/13.5.7) is not vulnerable:

> Bug 43201: Security fixes from Firefox 131.0.2 [tor-browser]

**Comment by tmo1** [comment]
Thank you for the correction. My Mullvad Browser's "About" window had declared it to be "13.5.7 (based on Mozilla Firefox 115.16.0esr)", and Mozilla's matrix had called 115.16.0 ESR vulnerable, so I panicked and filed this issue, but upon further reading I had started to realize that Tor / Mullvad had apparently cherry picked the security fix and applied it to their version of 115.16.0 ESR. Sorry for the noise (but you might consider adding a clarification and / or documenting this sort of thing, e.g. "13.5.7 (based on Mozilla Firefox 115.16.0esr, with security fixes from 131.0.2)").
